docs/devguide.md

```diff
@@ -297,6 +298,36 @@ If you choose third party resources storage, the helm chart will not launch an
 etcd server, but will instead instruct the API server to store all resources in
 the Kubernetes cluster as third party resources.
 
+## Dependency Management
+We use [dep](https://golang.github.io/dep) manage our dependencies. We commit the resulting
+vendor directory to ensure repeatable builds and isolation from upstream source disruptions.
+Because vendor is committed, you do not need to interact with dep unless you are
```

Comment:
Unrelated question: do we run `dep ensure` (and check that the exit code is `0`) as part of the build to make sure that `vendor` directory is up to date with `Gopkg.lock`?

Reply:
nit:  We use dep .. *to* manage  (missing the word "to")

Reply:
Yup! That's handled in the `verify-vendor` target, which is invoked during `verify`.